data/super_secret_password_file.yaml

```diff
@@ -0,0 +1 @@
+top_secret_password: 123456789
```

Comment:
You should not write the top secret password like this. I am disappointed coach. I taught you better 😆